src/bashly.yml

```diff
@@ -42,6 +41,26 @@ commands:
         required: true
     examples:
       - dcu logo coreboot.rom -l bootsplash.bmp
+  - name: smmstore
```

Comment:
We may update it accordingly if we come up with a better naming (see below)

Reply:
I'd change it to `variable`

Reply:
https://github.com/Dasharo/dcu/pull/5/commits/2cf531f645fc532b64383b6104c3be1b6fba0c75